.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@7.x
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
+      - name: Webhint Report
+        run: npx hint .
+  stylelint:
+    name: Stylelint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check node_modules existence
+        run: |
```

Comment:
- [ ] Great job on adding the linters file but you need also to add the files for `hintrc` to check for HTML linters and `stylelint` to check for CSS linters so kindly add them to your project 


![image](https://user-images.githubusercontent.com/58553711/197897397-4ab83e38-1728-4992-924e-f9e26e1080ec.png)

## IF you scroll dow you will see the commands so you can check if there's any linters error need to fix 💯☺💪🏻
- This Command to check for HTML linters errors
![image](https://user-images.githubusercontent.com/58553711/197897533-9fca57f9-2f2b-4e38-ba1f-eb914601918c.png)

- This Command to check for CSS linters errors 💯
![image](https://user-images.githubusercontent.com/58553711/197897653-3182476c-2a50-48b4-bab7-3cebd112c3d1.png)

Reply:
Thanks for the feedback.
Where shall I add these files, under the root folder or .github/workflows?

Reply:
@Reem-lab Thanks for the feedback.
Where shall I add these files, under the root folder or .github/workflows?